src/API/Reports/PerformanceIndicators/Controller.php

```diff
@@ -42,6 +42,13 @@ class Controller extends \WC_REST_Reports_Controller {
 	 */
 	protected $endpoints = array();
 
+	/**
+	 * Contains a list active Jetpack module slugs.
+	 *
+	 * @var array
+	 */
+	protected $active_jetpack_modules = null;
```

Comment:
Really like the caching of the modules, nice.

Reply:
Agreed, seems like it gets hit ±4 times per request, so caching makes sense 👍